### ISSUE TITLE
[SpamHaus BCL] Create the connector

### ISSUE THREAD

**Issue by nicolasleguillou** [OpenCTI-Platform/connectors#378]
Please replace every line in curly brackets { like this } with appropriate answers, and remove this line.

## Problem to Solve

Create a SpamHaus BCL import connector: https://www.spamhaus.org/faq/section/Spamhaus%20BCL

**Comment by SamuelHassine** [comment]
@nicolasleguillou The access to the SpamHaus BCL is restricted. Can you please provide us an access (via Slack or email) so we can work on the connector? Thanks!

**Comment by nicolasleguillou** [comment]
Yes indeed! I will get back to you @SamuelHassine with an access as soon as possible.

**Comment by SamuelHassine** [comment]
Any news on the access to the API @nicolasleguillou?

**Comment by nicolasleguillou** [comment]
Yes @SamuelHassine, we can close this issue. In the end, the partner did not provide us with an access.

**Comment by SamuelHassine** [comment]
Thanks @nicolasleguillou, I keep this issue open as it is an interesting connector anyway.
